templates/commands/goldentest/test_funcs.go

```diff
@@ -52,33 +52,35 @@ const (
 )
 
 // parseTestCases returns a list of test cases to record or verify.
-func parseTestCases(ctx context.Context, location, testName string) ([]*TestCase, error) {
+func parseTestCases(ctx context.Context, location string, testNames []string) ([]*TestCase, error) {
 	if _, err := os.Stat(location); err != nil {
 		return nil, fmt.Errorf("error reading template directory (%s): %w", location, err)
 	}
 
 	testDir := filepath.Join(location, goldenTestDir)
 
-	if testName != "" {
-		testConfig := filepath.Join(testDir, testName, configName)
-		test, err := parseTestConfig(ctx, testConfig)
-		if err != nil {
-			return nil, err
-		}
-		return []*TestCase{
-			{
+	testCases := []*TestCase{}
+
+	if len(testNames) > 0 {
+		for _, testName := range testNames {
+			testConfig := filepath.Join(testDir, testName, configName)
```

Comment:
This block of code appears twice, here and at line 89. Can it be factored out to avoid duplication?

Reply:
Done. Did not save lines but looks cleaner now :)